core/domain/contributor_admin_dashboard_services.py

```diff
@@ -0,0 +1,458 @@
+# Copyright 2023 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Funtions to get stats Displayed in contributor Admin Dashboard.
```

Comment:
We don't name services files based on the pages instead entity names are used like exp_services, story_services. I would suggest either moving these functions to suggestion_services (As all the models are stored in suggestion_models.py) or renaming it to suggestion_stats_services/contribution_stats_services.py

Reply:
@kshitij01042002 is this resolved?

Reply:
Yeah. I have renamed the files to contribution_stats_services.py